Sources/OpenTelemetrySdk/Resources/ResourceAttributes.swift

```diff
@@ -1170,48 +1170,47 @@ public enum ResourceAttributes: String {
         /**
         cpp.
         */
-        static let cpp = TelemetrySdkLanguageValues("cpp")
+        public static let cpp = TelemetrySdkLanguageValues("cpp")
         /**
         dotnet.
         */
-        static let dotnet = TelemetrySdkLanguageValues("dotnet")
+        public static let dotnet = TelemetrySdkLanguageValues("dotnet")
         /**
         erlang.
         */
-        static let erlang = TelemetrySdkLanguageValues("erlang")
+        public static let erlang = TelemetrySdkLanguageValues("erlang")
         /**
         go.
         */
-        static let go = TelemetrySdkLanguageValues("go")
+        public static let go = TelemetrySdkLanguageValues("go")
         /**
         java.
         */
-        static let java = TelemetrySdkLanguageValues("java")
+        public static let java = TelemetrySdkLanguageValues("java")
         /**
         nodejs.
         */
-        static let nodejs = TelemetrySdkLanguageValues("nodejs")
+        public static let nodejs = TelemetrySdkLanguageValues("nodejs")
         /**
         php.
         */
-        static let php = TelemetrySdkLanguageValues("php")
+        public static let php = TelemetrySdkLanguageValues("php")
         /**
         python.
         */
-        static let python = TelemetrySdkLanguageValues("python")
+        public static let python = TelemetrySdkLanguageValues("python")
         /**
         ruby.
         */
-        static let ruby = TelemetrySdkLanguageValues("ruby")
+        public static let ruby = TelemetrySdkLanguageValues("ruby")
         /**
         webjs.
         */
-        static let webjs = TelemetrySdkLanguageValues("webjs")
+        public static let webjs = TelemetrySdkLanguageValues("webjs")
         /**
-         swift.
-         */
-        static let swift = TelemetrySdkLanguageValues("swift")
-
+        swift.
+        */
+        public static let swift = TelemetrySdkLanguageValues("swift")
```

Comment:
It's worth noting that this is something that we have added to the attributes file ourselves. If you regenerate, it will go.

Now that I'm using this value within the resource attributes, at least our tests will fail though so we should be reminded to add it back manually.

Reply:
Yes, I also added it to the spec and it has been merged since then, so when the spec version is updated the value will be automatically generated